include/pybind11/docstring_options.h

```diff
@@ -0,0 +1,73 @@
+/*
+    pybind11/docstring_options.h: selectively suppress the automatic generation of docstrings and function signatures.
+
+    Copyright (c) 2016 Wenzel Jakob <wenzel.jakob@epfl.ch>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#pragma once
+
+#include "common.h"
+
+NAMESPACE_BEGIN(pybind11)
+
+class docstring_options
+{
```

Comment:
Very minor, but: could you try using consistent braces indentation throughout the file? (there's at least three different ways here). 

This one is perhaps the most popular in the current codebase:

```cpp
foo {
    bar
}
```